lib/ansible/modules/system/mount.py

```diff
@@ -229,7 +229,11 @@ def set_mount(module, args):
             ) = line.split()
 
         # Check if we found the correct line
-        if ld['name'] != escaped_args['name']:
+        if (ld['name'] != escaped_args['name'] or
```

Comment:
Please format it like this:

```python
        # Check if we found the correct line
        # (swap is a special case because it always uses 'none' as its name)
        if (
                ld['name'] != escaped_args['name'] or (
                    ld['src'] != escaped_args['src'] and
                    escaped_args['fstype'] == 'swap')):
```